### PR TITLE
Story 21

### PR DESCRIPTION
**Pull request by ENoll78**
Passed unit tests in eclipse; double-check SecurityConfiguration.java. Not sure if perms are done correctly for frontend. CollegeAdmin is bare until we figure out what else to add. Phone#? Campus Office?

### CODE REVIEW

File: src/main/java/com/collegecompendium/backend/controllers/CollegeAdminController.java [hunk 1]
```diff
@@ -105,47 +106,52 @@ public College modifyCollege(
...
-			}
+
+	    Optional<College> collegeQuery = collegeRepository.findById(collegeId);
+        if (collegeQuery.isEmpty()) {
+        	response.setStatus(404);
+            return null;
+        }
+		List<CollegeAdmin> collegeAdmins = collegeAdminRepository.findByCollege(collegeQuery.get());
+		if (collegeAdmins.isEmpty()) {
+			response.setStatus(204);
```
Comment (meta1203) [inline src/main/java/com/collegecompendium/backend/controllers/CollegeAdminController.java]: 404, not 204

File: src/main/java/com/collegecompendium/backend/repositories/CollegeAdminRepository.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+// this interface extends CrudRepository, which uses the generic types
+// CollegeAdmin (denotes the class to provide a CRUD repo for) and String
+// (denotes the ID type on the class)
+public interface CollegeAdminRepository extends CrudRepository<CollegeAdmin, String>{
+	// so, this doesn't actually need any defined functions to get started
+	// on a basic level. If you need more than just "get by ID" (which you likely
+	// will at some point), check out this web page for info on how to do that:
+	// https://docs.spring.io/spring-data/data-jpa/docs/current/reference/html/#jpa.query-methods.query-creation
+	
+	//List<CollegeAdmin> findByDegreesIn(Degree... degrees);
```
Comment (meta1203) [inline src/main/java/com/collegecompendium/backend/repositories/CollegeAdminRepository.java]: the `findByDegrees...` functions should be moved to the CollegeRepository.

**Review by meta1203 — CHANGES_REQUESTED** [review]
make those two changes and this'll be good to go

**Comment by ENoll78** [comment]
Should be done

**Review by meta1203 — APPROVED** [review]
looks good, need to perform merge to get into main

**Pull request merged**